batches.py

```diff
@@ -30,46 +30,51 @@ def make_training_batch(start_index):
 
     for idx in range(start_index, end_index):
         tokens_a_index, tokens_b_index = idx, idx + 1
-        tokens_a, tokens_b = token_list[tokens_a_index], token_list[tokens_b_index]  # grab sentence pair
+        tokens_a = token_list[tokens_a_index]
```

Comment:
what is tokens_a and tokens_b? Can we put some comments in here explaining it... its a bit of a headache looking at without context

Reply:
is this the NSP section?

Reply:
i.e. are we doing masked word prediction as well as NSP?